src/client/dfs/duns.c

```diff
@@ -1304,11 +1307,33 @@ duns_link_cont(daos_handle_t poh, const char *cont, const char *path)
 		}
 		D_GOTO(err_link, rc);
 	}
+	if (backend_dfuse) {
+		struct stat finfo;
+		/*
+		 * This next stat will cause dfuse to lookup the entry point and perform a
+		 * container connect, therefore this data will be read from root of the new
+		 * container, not the directory.
+		 *
+		 * TODO: This could call getxattr to verify success.
+		 */
+		rc = stat(path, &finfo);
+		if (rc) {
+			rc = errno;
+			D_ERROR("Failed to access new container: %d (%s)\n", rc, strerror(rc));
```

Comment:
I know this is moved, not new code but it would be good to use the correct macros here.  The `S` here stands for system, that is it's a errno.
```suggestion
			DS_ERROR(rc, "Failed to access new container");
```

Reply:
- [ ] Use up to date logging macros

Reply:
> * [x]  Use up to date logging macros

Fixed with commit 0561a5b6c